examples/player/simid_player.html

```diff
@@ -23,15 +23,30 @@
     }
   </style>
   <script>
+    //if ad is non-linear allow user to insert specific dimensions
+    function enableDisable() {
+      let isNonLin = document.getElementById("nonlinear_ad");
+      let uniqueDimensions = document.querySelectorAll(".unique_dimension");
+      uniqueDimensions.forEach(function(dimension) {
+        if (isNonLin.checked) {
+          dimension.disabled = false;
+        }
+        else {
+          dimension.disabled = true;
+        }
+      });
+    }
+
     let simidPlayer;
     function initAd() {
       // clean old ad if it exists
       if (simidPlayer) {
         simidPlayer.stopAd();
       }
 
+      let isLinear = document.getElementById("linear_ad").checked;
```

Comment:
I'm worried you added to this PR stuff that's in kaleabs pr.

Are you using
git commit -a -m "stuff"

Prefer using
git add file_I_want_to_add.js
git commit -m "a comment about this commit"


That way files you don't want to be part of the PR won't be there.

Reply:
it's because we are branching off of each others. Will try to fix and compare to his PR not master